oeps/oep-0027-course-metadata-authoring.rst

```diff
@@ -0,0 +1,151 @@
+| Type            | Architecture                                                   |
++-----------------+----------------------------------------------------------------+
+| Created         | 2018-07-13                                                     |
++-----------------+----------------------------------------------------------------+
+| `Review Period` | 2018-08-01 - 2018-08-15                                        |
++-----------------+----------------------------------------------------------------+
+
+Context
+-------
+
+An understanding of `edX subdomains`_ is a prerequisite for this OEP.
+
+In the Open edX system, course content authoring occurs in the Studio tool, which focuses mainly on authoring of
+course structure and content within a **Course Run**. A Course Run is discoverable on the edX marketing site via
+the context of a **Catalog Course**. The **Catalog Course** data authoring needs a good user experience.
+
+Today, Catalog Course metadata authoring occurs in multiple places, including the Studio, Ecommerce, and
+Marketing sites. We introduced a new tool, called Publisher, to ease the Catalog Course metadata 
+authoring process. The Publisher tool includes an inflexible workflow engine for the process of provisioning
+a course. Currently, this course publish cycle can take up to fifteen days in order to provision 
+a new course. 
+
+We also run into data synchronization issues between the separate services since there are multiple sources of truth and
+multiple writers of the relevant metadata.
+
+This project will leverage edX's `decoupled frontend architecture`_ plan. In particular, we will be using phase 3 of the plan to separate
+the Content Metadata Authoring frontend from the associated backend.
+
+  .. _decoupled frontend architecture: https://openedx.atlassian.net/wiki/spaces/FEDX/pages/790692200/Decoupled+Frontend+Architecture
+  .. _edX subdomains: https://openedx.atlassian.net/wiki/spaces/AC/pages/663224968/edX+DDD+Bounded+Contexts#edXDDDBoundedContexts-edXSubdomainMap
+
+
+Decisions
+---------
+
+* We will no longer embed workflow into the core system. Rather, we will develop or enhance existing APIs to offload workflow and business process to systems and services that allow for more agile changes.
+
+* Conceptually, we will separate the notion of Course within the edX system in the following ways:
+
+  * **Course Learning Content**, a course-team created package of learning material, with its weekly and sub-weekly structure
+    (i.e., Sections, Subsections, Units, etc.) and learning blocks (i.e., leaf xBlocks), tied to specific dates of offering.
+     
+    * This is what we call **(Studio) Course Run**, for the time being.
+    * It is bound to the organization that authors the content.
+    * Note: In the future, this concept may be further separated when we support content reuse across "Course Runs".
+      At that time, we may distinguish between:
```

Comment:
What use case / feature is this note driving toward? I'm not sure what the goal is here. Would this replace the underlying behavior of the "re-run course" interaction?